### PR TITLE
gitignore - ignore *.swp and .idea

### PR DESCRIPTION
**Pull request by ondrejsika**
Ignore VIM's temporary files `*.swp` and JetBrains's `.idea`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@ MANIFEST
 .DS_Store
 _book
 node_modules
+.idea
+.swp
```
Comment (ekohl) [inline .gitignore]: Doesn't this need to be ```*.swp```?

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,6 @@ MANIFEST
...
+
```
Comment (ekohl) [inline .gitignore]: Is this empty line needed?

File: .gitignore (hunk 1 above)
Comment (ondrejsika) [reply .gitignore]: It can be `.*.swp` but I think that is alright.

File: .gitignore (hunk 2 above)
Comment (ondrejsika) [reply .gitignore]: It's recommended by Git

Comment (ekohl) [reply .gitignore]: Do you have a source for this? https://git-scm.com/docs/gitignore doesn't mention anything.

Comment (ondrejsika) [reply .gitignore]: oh, sorry, i made a mistake, it doesn't matter.

Comment (ekohl) [reply .gitignore]: Could you remove this?

Comment (ondrejsika) [reply .gitignore]: removed https://github.com/ondrejsika/djangogirls-tutorial/commit/c67bebc9f8a11294ae8019bc0d30fa42dcb5c87b

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**